test/testdata/array-errors.json

```diff
@@ -3,7 +3,7 @@
 		"lineNumber": "1",
 		"properties": {
 			"recordID": "1",
-			"filler": " "
+			"filler": ""
```

Comment:
That would be correct based on the fact that we now `.trim()` the fields in the HMDA JSON during processing.